### DISCUSSION TITLE
Welcome to source-engine Discussions!

### DISCUSSION THREAD

**Discussion by nillerusr** [nillerusr/source-engine#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KainAlvaine** [comment]
Hi ! Thank you very much for this glorious release ! But I got a little little problem! An graphic artifact! Small error! Got the classic Half life 2 too Dark in Linux! And no command or option of brightness and contrast can change it ! Hope you're doing fine and wish you luck and hope you can at least reply something to me ! God luck!

**Comment by BoxIam** [comment]
Hey I was wondering if you could make it port games over to switch?

**Comment by LegendaryGuard** [comment]
Wouldn't it be nice to have documentation? There are incomplete stuff when comes to build. 
It would be cool to write a mini-tutorial about adding a command that does something, or prints something to the screen (some text or whatever), if the contributors are starting to touch the engine code.

Also, communications will be through chat or here or both?

**Comment by light-and-ray** [comment]
У меня может глупый вопрос, но зачем основывать порт на слитом коде, когда есть официальный. https://github.com/ValveSoftware/source-sdk-2013  На сколько я понимаю, source-sdk - это отвязанный от hl2 движок. Я и когда-то на нем что-то запускал, выглядело как мод для hl2 без ресурсов hl2, да и исходные файлы на первый взгляд те же самые, есди взглянуть на имена файлов

Дело только в том, что слитый код 2017 года? Или есть более существенная разница? Конечно Valve - не Nintendo и не TakeTwo, фанатов не преследуют (кроме слива 2003 года, перед релизом hl2). Но все равно странновато, я думал исходный код source engine в сливах не нуждается

P.s. за работу большое спасибо, вопрос просто из любопытства )

**Reply by nillerusr** [reply]
@light-and-ray это не полный исходный код, это только часть его, достаточная для создания модификаций. Тут нету исходного когда самого движка, физики и т.д.

**Comment by ege-del** [comment]
Can you please explain how you generated missing .vpc files ? most of the leaks seem to be missing some .vpc files which causes project generation tools to fail, I tried running the bins in devtools directory but I could not figure out the right order..

**Reply by nillerusr** [reply]
I am not using vpc anymore. Please read wiki

**Comment by zaczolp** [comment]
Wanted to say thanks for putting this together. Works amazingly well on Apple silicon

**Reply by zaczolp** [reply]
Only issue I had was not being able to tab away from the game and a graphical glitch when using the flashlight, apart from that its great

**Comment by tuco298** [comment]
Is there a way to put texts in touch.cfg? Or is not possible
